inyoka_theme_default/templates/planet/index.html

```diff
@@ -77,15 +82,24 @@
                     <div class="text">{{ article.text }}</div>
                   </div>
                 </div>
-                <div class="planet-article-footer">
-                  <a href="{{ article|url|e }}">{% trans %}Permanent link{% endtrans %}</a>
+
+                <ul class="planet-article-footer">
```

Comment:
1. 
2. no it doesn't because you only replace the div with the ul and add an `<li>` for each element. So actually you have more markup.
3. It depends on your point of view. ;)